README.md

```diff
@@ -16,91 +16,43 @@ Once Docker is installed, run this to bootstrap the containers:
 $ docker-compose up --no-start
 ```
 
-Then run this to install the client-side and functions dependencies
-respectively:
-
-```sh
-$ docker-compose run --rm app yarn
-$ docker-compose run --rm functions yarn --cwd functions
-```
-
-Finally, you will need to add your Firebase credentials to the local
+You will need to add your Firebase credentials to the local
 project configuration. If you have not created a Firebase app to use for
 development, [do that now](https://console.firebase.google.com/).
```

Comment:
A couple of things I ran into the first time - I'm not sure if they are addressed here. 
1. Enabling google auth on firebase console
2. Creating the firebase firestore database
3. Making sure that the permissions on the database are appropriate. Should it be set to test?

Reply:
Good call—was thinking too narrowly about what you need to do to get set up within the repo.